### PR TITLE
Scarab fragments rebased

### PR DESCRIPTION
**Pull request by aydjay**
Rebased after previous PR attempt. I closed #909  to avoid confusion.

Update the Fragment stash tab background and positions

Add the Scarabs to the Fragment Stash tab viewmodel.

Before I continue with the scarabs I need some more info @thailyn:

Rank placements:
Rusted, Polished, Gilded (Left to right)
Effect Placements:
Which effect where!

If anyone has a fragment tab with some of each grade of scarab i would be very appreciative as:
* I don't have a fragment stash tab
* I don't have a single scarab yet! (Still on lvl 50 on one char)

### CODE REVIEW

File: Procurement/Controls/FragmentStashViewModel.cs [hunk 1]
```diff
@@ -49,18 +48,63 @@ public FragmentStashViewModel(List<Item> items)
         public ItemDisplayViewModel EshBlessing => new ItemDisplayViewModel(GetBreach<BreachBlessing>(BreachType.Esh));
         public ItemDisplayViewModel TulBlessing => new ItemDisplayViewModel(GetBreach<BreachBlessing>(BreachType.Tul));
         public ItemDisplayViewModel UulNetolBlessing => new ItemDisplayViewModel(GetBreach<BreachBlessing>(BreachType.UulNetol));
+
         public ItemDisplayViewModel XophBlessing => new ItemDisplayViewModel(GetBreach<BreachBlessing>(BreachType.Xoph));
 
+        public ItemDisplayViewModel GildedAmbushScarab => new ItemDisplayViewModel(GetScarab(ScarabRank.Gilded, ScarabEffect.Ambush));
```
[the remark this answers is not in the document]
Comment (aydjay) [reply Procurement/Controls/FragmentStashViewModel.cs]: I'm leaving the line length as is. The vast majority of programmers use widescreens now and these lines fit amply for me. 

I agree that when there are long slabs of logic in if statements it's pertinent to keep the lines readable but for a get only property in a large wedge of code like this, having the formatting the way it is currently suits me.

File: POEApi.Model/Scarab.cs [hunk 2]
```diff
@@ -1,15 +1,127 @@
...
+            ScarabEffect = GetScarabEffect();
+        }
+
+        public ScarabRank ScarabRank { get; }
+
+        public ScarabEffect ScarabEffect { get; }
+
+        private ScarabEffect GetScarabEffect()
+        {
+            if (TypeLine.Contains("Ambush"))
```
[the remark this answers is not in the document]
Comment (aydjay) [reply POEApi.Model/Scarab.cs]: I will improve the handling here.

**Comment by Stickymaddness** [comment]
@aydjay Damnit, I forgot this one was still open and merged your other two PRs, creating conflicts on this one again :(

**Comment by thailyn** [comment]
If you do merge earlier rather than later, feel free to open an issue to confirm the scarab placements are correct and assign it to me.

**Pull request merged**